### PR TITLE
Update Minor Version Maven Dependencies

### PR DESCRIPTION
**Pull request by thewriteway**
List of version changes and success status:
image
Regression fixes included.

### CODE REVIEW

**Comment by thewriteway** [comment]
Hi Kagemonji, I tried to update as many minor dependencies as possible in this PR.  I cant tell if the build tests are too sensitive or my PR code isn't very compatible.

File: pom.xml [hunk 1]
```diff
@@ -371,6 +387,9 @@
                                 <ignoredUnusedDeclaredDependency>org.apache.cxf:cxf-tools-wsdlto-databinding-jaxb</ignoredUnusedDeclaredDependency>
                                 <ignoredUsedUndeclaredDependency>org.glassfish.jaxb:jaxb-runtime</ignoredUsedUndeclaredDependency>
                             </ignoredUnusedDeclaredDependencies>
+    <ignoredNonTestScopedDependencies>
+<ignoredNonTestScopedDependency> org.springframework:spring-aspects</ignoredNonTestScopedDependency>
+                            </ignoredNonTestScopedDependencies>
```
Comment (kagemomiji) [inline pom.xml:392]: It seems unnecessary definition.
```suggestion
```

**Review by kagemomiji — CHANGES_REQUESTED** [review]
@thewriteway 

Thank you for submitting the updates.

Upon reviewing the CI errors, it appears to be an issue related to logback dependencies, so I believe that changing the logback version is the direct cause of the error. 
I have also commented on other changes that need to be made.

**Review by kagemomiji — APPROVED** [review]
LGTM

**Pull request merged**